### PR TITLE
Add Canon EOS Utility v3.8.20 cask

### PR DESCRIPTION
**Pull request by teran**
Signed-off-by: Igor Shishkin <me@teran.ru>

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-drivers/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/canon-eos-utility.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                      '/Applications/Canon Utilities/CameraSurveyProgram',
+                      '/Applications/Canon Utilities/EOS Lens Registration Tool',
+                      '/Applications/Canon Utilities/EOS Utility',
+                      '/Applications/Canon Utilities/EOS Web Service Registration Tool',
+                      '~/Library/Caches/com.canon.EOS-Lens-Registration-Tool',
+                      '~/Library/Caches/com.canon.UpdateChecker',
+                      '~/Library/Preferences/jp.co.canon.ic.camerasurveyprogram.plist',
+                      '~/Library/Preferences/com.canon.EOS Utility 2.plist',
+                      '~/Library/Preferences/com.canon.EOS-Utility.plist',
+                      '~/Library/Preferences/com.canon.EOS-Utility-3.plist',
```
Comment (reitermarkus) [inline Casks/canon-eos-utility.rb]: The preferences should be in `zap`.

**Comment by teran** [comment]
@reitermarkus your comment is fixed, thanks for review.

File: Casks/canon-eos-utility.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  installer manual: "eum#{version}-installer.app"
+
+  uninstall delete: [
+                      '/Applications/Canon Utilities/Auto Update Service',
+                      '/Applications/Canon Utilities/CameraSurveyProgram',
+                      '/Applications/Canon Utilities/EOS Lens Registration Tool',
+                      '/Applications/Canon Utilities/EOS Utility',
+                      '/Applications/Canon Utilities/EOS Web Service Registration Tool',
+                    ]
```
Comment (reitermarkus) [inline Casks/canon-eos-utility.rb]: Also add `rmdir: '/Applications/Canon Utilities'.

Comment (teran) [reply Casks/canon-eos-utility.rb]: Done.

**Comment by reitermarkus** [comment]
Thanks, @teran!

**Pull request merged**